python/taichi/_lib/utils.py

```diff
@@ -142,10 +142,33 @@ def require_version(major, minor=None, patch=None):
 at_startup()
 
 
+def compare_version(latest, current):
+    latest_num = map(int, latest.split('.'))
+    current_num = map(int, current.split('.'))
+    for x, y in zip(latest_num, current_num):
```

Comment:
The `tuple[int]` can be compared directly with `<`/`>`/`==`

Reply:
Thanks for the info!